openapps/opencoap/opencoap.c

```diff
@@ -846,19 +873,23 @@ owerror_t opencoap_options_encode(
     return E_SUCCESS;
 }
 
-coap_option_iht* opencoap_find_option(coap_option_iht* array, uint8_t arrayLen, coap_option_t option) {
-    uint8_t i;
-
-    if (array == NULL || arrayLen == 0) {
-        return NULL;
-    }
+uint8_t opencoap_find_option(coap_option_iht* array, uint8_t arrayLen, coap_option_t option, uint8_t* startIndex) {
+   uint8_t i, j;
```

Comment:
In OpenWSN, we typically declare one variable per line. Could you fix this throughout the code?